src/cbmc_starter_kit/template-for-ci-workflow/proof_ci.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+name: Run CBMC proofs
+on:
+  push:
+    branches-ignore:
+      - gh-pages
+
+# USAGE
+#
+# If you need to use different versions for tools like CBMC, modify this file:
+# .github/workflows/proof_ci_resources/config.yaml
+#
+# If you want the CI to use a different GitHub-hosted runner (which must still
+# be running Ubuntu 20.04), modify the value of this key:
+# jobs.run_cbmc_proofs.runs-on
+
+jobs:
+  run_cbmc_proofs:
+    runs-on: <__GITHUB_ACTIONS_RUNNER__>
+    name: run_cbmc_proofs
+    permissions:
+      contents: read
+      id-token: write
+      pull-requests: read
+    steps:
+      - name: Check out repository and submodules recursively
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: Parse config file
+        run: |
+          CONFIG_FILE='.github/workflows/proof_ci_resources/config.yaml'
+          touch $CONFIG_FILE
```

Comment:
Sorry, my mistake in the example code in the previous review comment. This should actually be `touch $GITHUB_ENV` I think. (We already know `$CONFIG_FILE` exists). If `$GITHUB_ENV` doesn't already exist, then `>>` on line 37 fails, so I think we should touch it here.